react-app/src/features/package/package-activity/index.tsx

```diff
@@ -374,9 +373,9 @@ export const PackageActivities = () => {
         className="flex flex-col gap-2"
         defaultValue={hook.accordianDefault}
       >
-        {hook.data?.map((CL) => (
-          <PackageActivity {...CL._source} key={CL._source.id} />
-        ))}
+        {hook.data?.map((CL) => {
+          return <PackageActivity {...CL._source} key={CL._source.id} />;
+        })}
```

Comment:
```suggestion
         {hook.data?.map((CL) => (
          <PackageActivity {...CL._source} key={CL._source.id} />
        ))}
```
Let's go back to this